src/earn/PoolCard.tsx

```diff
@@ -18,17 +20,45 @@ import { Spacing } from 'src/styles/styles'
 import { tokensByIdSelector } from 'src/tokens/selectors'
 import { TokenBalance } from 'src/tokens/slice'
 
-export default function PoolCard({ pool, testID = 'PoolCard' }: { pool: Pool; testID?: string }) {
-  const { tokens, networkId, poolTokenId, depositTokenId } = pool
+export default function PoolCard({
+  pool,
+  testID = 'PoolCard',
+}: {
+  pool: EarnPosition
+  testID?: string
+}) {
+  const {
+    tokens,
+    networkId,
+    dataProps: { withdrawTokenId: poolTokenId, depositTokenId },
+  } = pool
   const { t } = useTranslation()
   const allTokens = useSelector((state) => tokensByIdSelector(state, [networkId]))
   const tokensInfo = useMemo(() => {
     return tokens
-      .map((tokenId) => allTokens[tokenId])
+      .map((token) => allTokens[token.tokenId])
       .filter((token): token is TokenBalance => !!token)
   }, [pool.tokens, allTokens])
-  const poolTokenInfo = allTokens[poolTokenId]
   const depositTokenInfo = allTokens[depositTokenId]
+
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const poolBalance =
+    useDollarsToLocalAmount(new BigNumber(pool.balance).times(new BigNumber(pool.priceUsd))) ?? null
+  const poolBalanceString = useMemo(
+    () => `${localCurrencySymbol}${poolBalance ? formatValueToDisplay(poolBalance) : '--'}`,
+    [localCurrencySymbol, poolBalance]
+  )
+
+  const rewardPercentageString = useMemo(
+    () =>
+      `${pool.dataProps.earningItems.reduce((acc, earnItem) => acc.plus(new BigNumber(earnItem.amount)), new BigNumber(0)).toFixed(2)}%`,
```

Comment:
nit: do the `reduce` and string formatting in separate lines, probably fine just to have the string formatting outside the useMemo.

Also, use destructuring assignment to extract `earningItems` instead of doing `pool.dataProps.<>` everywhere